### PR TITLE
Remove linear erc721 voting

### PR DESCRIPTION
**Pull request by adamgall**
Please review and merge https://github.com/decentdao/fractal-interface/pull/1668 first.

### CODE REVIEW

File: src/hooks/DAO/proposal/useUserERC721VotingTokens.ts [hunk 1]
```diff
@@ -173,14 +179,18 @@ export default function useUserERC721VotingTokens(
...
               totalTokenAddresses.push(tokenAddress);
               totalTokenIds.push(tokenId);
               if (_proposalId) {
-                const tokenVoted = await votingContract!.hasVoted(
-                  _proposalId,
-                  tokenAddress,
-                  tokenId,
-                );
+                const tokenVoted = await votingContract.read.hasVoted([
+                  Number(_proposalId),
```
Comment (Da-Colon) [inline src/hooks/DAO/proposal/useUserERC721VotingTokens.ts:190]: Can proposal Id ever be anything but a number? possibly so? just wanted to ask anyway

**Review by Da-Colon — COMMENTED** [review]
👍

**Review by Da-Colon — APPROVED** [review]

File: src/hooks/DAO/proposal/useUserERC721VotingTokens.ts (hunk 1 above)
Comment (adamgall) [reply src/hooks/DAO/proposal/useUserERC721VotingTokens.ts:190]: Good call out. Found some bugs here. Fixing now.

Comment (adamgall) [reply src/hooks/DAO/proposal/useUserERC721VotingTokens.ts]: Well, fixed the bugs (5fade89d87a6765fbe6dbc8f58f5d0283f2270b5).

I also moved where this cast was happening, but TL;DR is that by the time this code here is executing, `_proposalId`, although a string, is a number. I certainly don't love this. The deeper issue is that we're sharing so many types/interfaces between erc20/erc721/multisig/snapshot proposals, when we really shouldn't be.

**Pull request merged**